src/pg/db/pool.js

```diff
@@ -0,0 +1,76 @@
+import { Pool } from 'pg';
+import { config as defaultConfig } from './config';
+
+import debug from 'debug';
+const log = (...text) => debug(`graffy:pg:${text.shift()}`)(text);
```

Comment:
```suggestion
const log = debug(`graffy:pg:pool`);
```

"error" is not a good tag for the debug library. The convention is to use tags to isolate the part of the application. The `debug` library that we use does not have a concept of log levels (error, warn, info).